packages/ui/src/lib/modal/Modal.svelte

```diff
@@ -6,12 +6,14 @@ import { tabWithinParent } from '../utils/dialog-utils';
 const dispatch = createEventDispatcher();
```

Comment:
createEventDispatcher is deprecated in svelte5 (ref https://svelte.dev/docs/svelte/v5-migration-guide#Event-changes-Component-events)

Reply:
I think there is like a deprecation policy where we introduced the callback onclose but if consumer are the event it's also working

so before deleting the dispatcher, it would require some time to replace the event by the callback on the extensions using the event